packages/turf-center-median/test.js

```diff
@@ -0,0 +1,46 @@
+import test from 'tape';
+import glob from 'glob';
+import path from 'path';
+import load from 'load-json-file';
+import write from 'write-json-file';
+import { featureCollection } from '@turf/helpers';
+import center from '@turf/center';
+import centerOfMass from '@turf/center-of-mass';
+import centerMean from '@turf/center-mean';
+import centerMedian from '.';
+
+test('turf-center-median', t => {
+    glob.sync(path.join(__dirname, 'test', 'in', '*.json')).forEach(filepath => {
+        // Define params
+        const {name} = path.parse(filepath);
+        const geojson = load.sync(filepath);
+        const properties = geojson.properties || {}
```

Comment:
The great thing now with the Optional parameters is you can simply just declare `options` as the GeoJSON Properties instead of declaring each parameter.

## Valid Options

```js
const options = geojson.properties;
```